src/lib/tenant/configs/contracts/scroll.ts

```diff
@@ -75,6 +76,14 @@ export const scrollTenantContractConfig = ({
       provider,
     }),
 
+    proposalTypesConfigurator: new TenantContract<BaseContract>({
```

Comment:
Nice ya, did the same in my branch here
https://github.com/voteagora/agora-next/pull/643

We will need a timelock too